content/Memory_Map.md

```diff
@@ -42,31 +42,52 @@ area are required to be specified correctly.
 
 # External Memory and Hardware
 
-The areas from 0000-7FFF and A000-BFFF may be used to connect external
-hardware. The first area is typically used to address ROM (read only, of
-course), cartridges with [Memory Bank Controllers
-(MBCs)](Memory_Bank_Controllers "wikilink") are additionally using this
-area to output data (write only) to the MBC chip. The second area is
-often used to address external RAM, or to address other external
-hardware (Real Time Clock, etc). External memory is
-often battery buffered, and may hold saved game positions and high score
-tables (etc.) even when the Game Boy is turned off, or when the
-cartridge is removed. For specific information read the chapter about
-Memory Bank Controllers.
+The areas from 0000-7FFF and A000-BFFF address external hardware, i.e.
+the carridge, which is essentially an expansion board.  Typically this
+is a ROM and SRAM or, more often, a [Memory Bank Controller (MBC)](Memory_Bank_Controllers "wikilink").
+The RAM area is accessed as RAM; writes to the ROM area control the
+MBC.  Some MBCs allow mapping of other hardware into the RAM area in
+this way.
+
+Cartridge RAM is often battery buffered to hold saved game positions,
+high score tables, and other information when the Game Boy is turned
+off.  For specific information read the chapter about Memory Bank
+Controllers.
 
 # Echo RAM
 
-The memory range E000-FDFF is a mirror (or \"echo\") of WRAM, both for
-reading and writing. For example, writing to \$E123 will modify both
-\$C123 and \$E123. It is recommended to avoid using this memory range
-anyways. This memory range's behavior has been confirmed on all grey
-GBs as well as on CGB and GBA. Some emulators (such as VisualBoyAdvance
-\<1.8) don't emulate Echo RAM. It is possible to check if Echo RAM is
-properly emulated by writing to WRAM (avoid values 00 and FF) and
-checking if said value is mirrored in Echo RAM.
+The range at E000-FDFF connects to WRAM, but only the lower 13 bits of
+the address lines are connected. with the upper bits on the upper bank
```

Comment:
The dot there was probably meant to be a comma; but this bit about the bank reg is unnecessary anyways, because what matter is that the addresses are considered equivalent. WRAMX banking occurs after that.